tests/php/Domain/Services/CreateAccount.php

```diff
@@ -98,130 +88,112 @@ public function test_create_customer_from_order( $email, $first_name, $last_name
 		$this->assertEquals( $test_order->get_customer_id(), $result['user_id'] );
 	}
 
-    public function create_customer_data()
-    {
-        return [
-        	// User requested an account.
-        	[
-        		'maryjones@testperson.net',
-        		'Mary',
-        		'Jones',
+	public function create_customer_data()
+	{
```

Comment:
nitpick: why is the bracket on a separate line? I know it's a PSR standard but we don't follow PSR (unfortunately).

Reply:
Aha! This was because I was pasting from PHPUnit docs!! Fixed: 9edaf7aff87f5814de2bbb374a7b62716c8b166e